core/frontend/src/LocalWebViewerApp.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module IModelApp
+ */
+
+import { IpcWebSocket, IpcWebSocketFrontend, IpcWebSocketMessage, IpcWebSocketTransport } from "@bentley/imodeljs-common";
+import { IpcApp } from "./IpcApp";
+
+class LocalTransport extends IpcWebSocketTransport {
```

Comment:
I don't quite get why we need to have a IpcWebSocket for web r/o cases.  I thought the intention was for Ipc to be used for r/w only.

Reply:
DTA is a test app which uses the standalone api, which needs ipc. Viewer apps in general will not use this.

Reply:
@calebmshafer DTA uses BriefcaseConnection so it can be used to test the editor apis. Otherwise it could just use RPC.

Reply:
Sounds good. I just didn't want people to start thinking this was the new way to write a r/o web app. I second what @kabentley  suggested about making this internal on the other comment.